HTML/mockup-mogo/index.html

```diff
@@ -0,0 +1,1024 @@
+                        incididunt ut labore et dolore magna aliqua. Ut enim ad minim.
+                    </p>
+                </div>
+            </div>
+            <div class="client-item">
+                <picture class="client-item__picture">
+                    <source media="(min-width: 800px)" srcset="img/layer-50@3x.png">
+                    <source media="(min-width: 500px)" srcset="img/layer-50@2x.png">
+                    <img src="img/layer-50.png" alt="man-picture">
+                </picture>
+                <div class="client-item__content">
+                    <h4 class="label">Jaelynn Castillo </h4>
+                    <p class="client-item__user paragraph paragraph--accordion">Graphic Design</p>
+                    <p class="paragraph">
+                        Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor
+                        incididunt ut labore et dolore magna aliqua. Ut enim ad minim.
+                    </p>
+                </div>
+            </div>
+            <div class="client-item">
+                <picture class="client-item__picture">
+                    <source media="(min-width: 800px)" srcset="img/layer-51@3x.png">
+                    <source media="(min-width: 500px)" srcset="img/layer-51@2x.png">
+                    <img src="img/layer-51.png" alt="man-picture">
+                </picture>
+                <div class="client-item__content">
+                    <h4 class="label">Mike Petrucci</h4>
+                    <p class="client-item__user paragraph paragraph--accordion">Graphic Design</p>
+                    <p class="paragraph">
+                        Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor
+                        incididunt ut labore et dolore magna aliqua. Ut enim ad minim.
+                    </p>
+                </div>
+            </div>
+        </div>
+    </section>
+
+    <section class="container section-blog">
+        <div class="section-clients__content">
+            <h4 class="subheading subheading--section">Our stories</h4>
+            <h3 class="heading heading--section">Latest blog</h3>
+        </div>
+        <div class="section-blog__blogs">
+            <div class="blog">
+                <picture class="blog__picture">
+                    <source media="(min-width: 800px)" srcset="img/layer-54@2x.png">
+                    <source media="(min-width: 500px)" srcset="img/layer-54@3x.png">
+                    <img src="img/layer-54@2x.png" alt="office-picture">
+                </picture>
+                <div class="blog__time">
+                    <h3 class="paragraph paragraph--big u-color-white">15</h3>
+                    <p class="paragraph paragraph--accordion u-color-white">Jan</p>
+                </div>
+                <h4 class="label">Lorem ipsum dolor sit amet </h4>
```

Comment:
usually it is a link also that user can go to this link and read full blog post, same applies for blog posts in footer